apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -498,29 +506,40 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     elif data["panel"] == "submission":
         if grading_method is GradingMethodType.EXTERNAL:
             return ""  # external grader is responsible for displaying results screen
+        feedback_type = pl.get_enum_attrib(
+            element, "feedback", FeedbackType, FEEDBACK_DEFAULT
+        )
+        show_first_wrong = feedback_type in [
+            FeedbackType.FIRST_WRONG,
+            FeedbackType.FIRST_WRONG_VERBOSE,
+        ]
+        student_submission = [
+            {
+                "inner_html": attempt["inner_html"],
+                "indent": (attempt["indent"] or 0) * TAB_SIZE_PX,
+                "badge_type": attempt["badge_type"] if show_first_wrong else "",
```

Comment:
Is the idea here to include this set of information only if `show_first_wrong` is set to true? If so, it would be better to add all these keys to the dictionary under a single `if show_first_wrong:` statement, instead of having a bunch of these inline if-else statements.

Reply:
I refactored to handle all of this conditional stuff in `grade` as it was before, some of it was being handled in `grade` and some here in `render`. Now its all in one place which I think is much cleaner.